docker/nginx/default.conf

```diff
@@ -1,18 +1,23 @@
 server {
     server_name tm.local;
-    root /var/www/html;
-    index index.php;
+    root /var/www/html/web;
 
-    location ~ \.php$ {
-        try_files $uri =404;
-        fastcgi_split_path_info ^(.+\.php)(/.+)$;
+    location / {
+        try_files $uri /index.php$is_args$args;
+    }
+
+    location ~ ^/(index|index_dev)\.php(/|$) {
         fastcgi_pass php:9000;
-        fastcgi_index index.php;
+        fastcgi_split_path_info ^(.+\.php)(/.*)$;
         include fastcgi_params;
         fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
-        fastcgi_param PATH_INFO $fastcgi_path_info;
+        fastcgi_param HTTPS off;
+    }
+
+    location ~ \.php$ {
+        return 404;
     }
 
-    error_log /var/log/nginx/tm.error.log;
-    access_log /var/log/nginx/tm.access.log;
-}
+    error_log /var/log/nginx/project_error.log;
```

Comment:
Not needed to change error path - my fault. I will improve this with next changes.